make_list_flat.py

```diff
@@ -0,0 +1,13 @@
+dim_list = [[[2]], [4, [5, 6, [6], 6, 6, 6], 7]]
+
+def flattern(array : list):
```

Comment:
you can also provide return type and docstring:


`
def flattern(array : list) -> list:
    '''
    This function handles smth
    '''
    
`